docs/reference/indices/index-templates.asciidoc

```diff
@@ -1,18 +1,12 @@
 [[indices-templates]]
-=== Index templates
+= Index templates
 ++++
-<titleabbrev>Index template</titleabbrev>
+<titleabbrev>Index templates</titleabbrev>
 ++++
 
-This documentation is about composable templates. For legacy templates please see the
-<<indices-templates-v1,legacy template documentation>>.
-
-[NOTE]
-====
-In {es} 7.8 composable templates were introduced. When a composable template matches a given index
-it always takes precedence over a legacy template. If no composable template matches, a legacy
-template may still match and be applied.
-====
+NOTE: This topic describes the composable templates introduced in {es} 7.8. 
+For information about how templates worked previously, 
+see the <<indices-templates-v1,legacy template documentation>>.
```

Comment:
We've now got a few different types of templates. I think specifying `index` is important for clarity here.
```suggestion
NOTE: This topic describes composable index templates introduced in {es} 7.8. 
For information about how templates worked previously, 
see the <<indices-templates-v1,legacy index template documentation>>.
```